CI.md

```diff
@@ -0,0 +1,475 @@
+# Setting up Cake Contrib Addins on CI providers
+
+<!-- markdownlint-disable -->
+<!-- TOC depthfrom:2 depthto:6 -->
+
+- [What is this](#what-is-this)
+- [Using AppVeyor](#using-appveyor)
+  - [Available Environment Variables](#available-environment-variables)
+  - [Using Cake.Recipe 2.x with AppVeyor](#using-cakerecipe-2x-with-appveyor)
+  - [Using Cake.Recipe 1.x with AppVeyor](#using-cakerecipe-1x-with-appveyor)
+    - [Additional notes](#additional-notes)
+- [Using Azure Pipelines](#using-azure-pipelines)
+- [Using GitHub Actions](#using-github-actions)
+  - [Available Organization Secrets](#available-organization-secrets)
+  - [Using Cake.Recipe 2.x with GitHub Actions](#using-cakerecipe-2x-with-github-actions)
+  - [Additional workflows](#additional-workflows)
+    - [Forcing documentation publishing](#forcing-documentation-publishing)
+    - [Drafting new releases](#drafting-new-releases)
+      - [Drafting new stable release](#drafting-new-stable-release)
+      - [Drafting new pre-release](#drafting-new-pre-release)
+
+<!-- /TOC -->
+<!-- markdownlint-enable -->
+
+## What is this
+
+The goal of this document is to help setting up new builds for different build
+providers under the cake-contrib organization.
+Some CI providers may already have Environment Variables or secrets set on the
+organization level that are available during the build.
+
+## Using AppVeyor
+
+### Available Environment Variables
+
+The following environment variables are accessible when running builds on the
+AppVeyor CI provider and will be available without further configuration.
+Private variables are noted, which means they will no be available in pull requests.
+However, these private variables can be made available in pull requests
+originating from the same repository.
+
+- `APPVEYOR_API_TOKEN` (_secret variable_)
+- `AZURE_PASSWORD`
+- `AZURE_SOURCE`
+- `AZURE_USER`
+- `GITHUB_PASSWORD` (_secret variable, can be used as a GitHub Token_)
+- `GITHUB_USERNAME`
+- `GITTER_ROOM_ID` (_secret variable_)
+- `GITTER_TOKEN` (_secret variable_)
+- `GPR_PASSWORD`
```

Comment:
And this? If it isn't, it should be :smile:

Reply:
```suggestion
- `GPR_PASSWORD` (_secret variable_)
```

Just adding so I can do a batch update (easier than to find them in vscode).